src/vm-repair/setup.py

```diff
@@ -8,7 +8,7 @@
 from codecs import open
 from setuptools import setup, find_packages
 
-VERSION = "1.1.0"
+VERSION = "1.2.0"
```

Comment:
As a minor change, it's better to upgrade the last version `1.1.1` instead of `1.2.0`

Reply:
@evelyn-ys , done.